lib/tasks/evm_dba.rake

```diff
@@ -271,6 +273,7 @@ namespace :evm do
           opt :uri,              "Destination depot URI",       :type => :string, :required => true
           opt :uri_username,     "Destination depot username",  :type => :string
           opt :uri_password,     "Destination depot password",  :type => :string
+          opt :uri_region,       "Destination depot region",    :type => :string
```

Comment:
Same here, would prefer if this could be `aws_region`

Reply:
No prob.  Will change.